<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchListDialog</class>
 <widget class="QDialog" name="BatchListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批次列表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>型号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modelComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="batchTypeLabel">
       <property name="text">
        <string>批次类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="batchTypeComboBox">
       <item>
        <property name="text">
         <string>正常批次</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>工序批次</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="batchLabel">
       <property name="text">
        <string>批次</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="batchComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="searchLabel">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>输入批次代码或关键字</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
   <widget class="QTableView" name="batchTableView"/>
  </item>
  <item>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>新增</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editButton">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>BatchListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
